samples/net/udp/boards/thingy91_nrf9160_ns.conf

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Disable the serial interface to reduce power consumption
+CONFIG_SERIAL=n
```

Comment:
This is enabled further down on line 18 in this same file

Reply:
Thanks, removed.